### PR TITLE
[backend/frontend] Background task 'delete all notifications' delete notifications for all users (#3285)

### LINKED ISSUE #3285: Background task 'delete all notifications' delete notifications for all users
## Description

Background task 'delete all notifications' delete notifications for all users

## Environment

1. OS (where OpenCTI server runs): { e.g. Mac OS 10, Windows 10, Ubuntu 16.4, etc. }
2. OpenCTI version: { e.g. OpenCTI 1.0.2 }
3. OpenCTI client: { e.g. frontend or python }
4. Other environment details:

## Reproducible Steps

- Log in as a user (user1)
- Log in as an other user (user2) in another window
- Submit to at least one trigger for each user (to have notifications for both users)
- As user1: select all the notifications and delete them via the toolbar (background task)
- All the notifications of user1 are deleted
- Problem: all the notifications of user2 are also deleted

## Expected Output

Only the notifications of user1 are deleted

## Additional information

The background task should take the user that delete his notifications into account.

## Screenshots (optional)
image

### PR DESCRIPTION
**Pull request by SouadHadjiat**
### Proposed changes

* Add missing user_id filter for notifications
* Filter entity types handled by background tasks (keep only stix objects and notifications)

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/3285

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: opencti-platform/opencti-front/src/private/components/profile/Notifications.tsx
Comment (Kedae) [inline opencti-platform/opencti-front/src/private/components/profile/Notifications.tsx]: useAuth()

File: opencti-platform/opencti-graphql/src/domain/task-common.js
Comment (Kedae) [inline opencti-platform/opencti-graphql/src/domain/task-common.js]: I Think you should use isStixCoreObject

**Comment by Archidoit** [comment]
Missing translation: filter_user_id
image

**Comment by Archidoit** [comment]
Behavior tested: OK 👍

**Comment by SouadHadjiat** [comment]
PR status : 
- Reviewed and tested by @Archidoit and @Kedae
- Review fixes pushed
- Waiting for final review and approval

**Review by Kedae — APPROVED** [review]

**Pull request merged**
